static/app/components/events/interfaces/crashContent/stackTrace/content.tsx

```diff
@@ -82,17 +169,16 @@ class Content extends Component<Props, State> {
     return penultimateFrame.inApp && !lastFrame.inApp;
   }
 
-  frameIsVisible = (frame: Frame, nextFrame: Frame) => {
-    const {includeSystemFrames} = this.props;
-
+  isRepeatedFrame(frame, nextFrame): boolean {
```

Comment:
One last thing! I noticed that the arguments here aren't typed. We should add them:

```
frame: Frame, nextFrame?: Frame
```